src/commands/sourcemaps/inject.rs

```diff
@@ -17,10 +17,11 @@ pub fn make_command(command: Command) -> Command {
             that id is used instead.",
         )
         .arg(
-            Arg::new("path")
-                .value_name("PATH")
-                .required(true)
-                .help("The path to the javascript files."),
+            Arg::new("paths")
+                .value_name("PATHS")
+                .num_args(1..)
+                .action(ArgAction::Append)
+                .help("Paths to the javascript files to inject."),
```

Comment:
```suggestion
                .help("A path to recursively search for javascript files that should be processed."),
```

Reply:
Using singular here, as help command shape indicate it allows for multiple values, and we use singular `path` in other places that behave the same way already.

Reply:
Good catch!